frontend/packages/console-app/src/components/pdb/PDBTableRow.tsx

```diff
@@ -26,15 +26,15 @@ const PodDisruptionBudgetTableRow: React.FC<RowProps<PodDisruptionBudgetKind>> =
       <TableData {...tableColumnInfo[0]} activeColumnIDs={activeColumnIDs}>
         <ResourceLink
           kind={referenceForModel(PodDisruptionBudgetModel)}
-          name={obj.metadata.name}
-          namespace={obj.metadata.namespace}
+          name={obj.metadata?.name || ''}
+          namespace={obj.metadata?.namespace || ''}
         />
```

Comment:
if we know PDBs will always have a name/namespace, can we modify the PDBKind instead?